ml/Host.h

```diff
@@ -84,8 +84,8 @@ class TrainableHost : public RrdHost {
     std::pair<Dimension *, Duration<double>> findDimensionToTrain(const TimePoint &NowTP);
     void trainDimension(Dimension *D, const TimePoint &NowTP);
 
+    struct rusage ResourceUsage{};
```

Comment:
I did not get the point about replacement of lines, just wondering the reason if the order effects something.